ci/docker/buildenv/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y git build-essential  autoconf flex bison \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)